test/framework/src/test/java/org/elasticsearch/test/VersionUtilsTests.java

```diff
@@ -195,7 +199,12 @@ public void testGradleVersionsMatchVersionUtils() {
                 /* Gradle skips the current version because being backwards compatible
                  * with yourself is implied. Java lists the version because it is useful. */
                 .filter(v -> v != Version.CURRENT)
-                .map(v -> v.major + "." + v.minor + "." + v.revision)
+                /* Note that gradle does *not* skip alphas, betas, or rcs here even
+                 * though we don't have backwards compatibility for alphas, betas,
```

Comment:
> I'm not sure why we don't have alphas in the Version.groovy. I mean, I know the regex doesn't match them, I just don't know why we treat them differently then betas

Because alphas are only cut from master, so there is never a case where we need to worry about bwc with an alpha. Only once we branch for a major release will we have betas/rcs.